ruby/ql/lib/codeql/ruby/dataflow/internal/DataFlowPrivate.qll

```diff
@@ -710,14 +710,16 @@ predicate clearsContent(Node n, Content c) {
   FlowSummaryImpl::Private::Steps::summaryClearsContent(n, c)
 }
 
-private newtype TDataFlowType = TTodoDataFlowType()
+private newtype TDataFlowType =
+  TTodoDataFlowType() or
+  TTodoDataFlowType2()
```

Comment:
This deserves a comment.  I would expect that this is also a problem for Python, where `DataFlowType` is also a singleton.

Reply:
Are you sure we shouldn't add a second constructor for Python as well?